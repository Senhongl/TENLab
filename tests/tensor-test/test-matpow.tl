a = [[1.1,2.0],[3.5,4.2]];
print(a^0);
print(a^1);
print(a^2);