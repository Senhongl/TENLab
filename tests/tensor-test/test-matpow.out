 1  0
 0  1
[ CPUDoubleType{2,2} ]
 1.1000  2.0000
 3.5000  4.2000
[ CPUDoubleType{2,2} ]
  8.2100  10.6000
 18.5500  24.6400
[ CPUDoubleType{2,2} ]
